Exception in thread "main" java.lang.ExceptionInInitializerError
	at java.base/sun.security.provider.PolicyFile.getKnownPermission(PolicyFile.java:889)
	at java.base/sun.security.provider.PolicyFile.getInstance(PolicyFile.java:839)
	at java.base/sun.security.provider.PolicyFile.addGrantEntry(PolicyFile.java:738)
	at java.base/sun.security.provider.PolicyFile.initDefaultPolicy(PolicyFile.java:491)
	at java.base/sun.security.provider.PolicyFile$2.run(PolicyFile.java:338)
	at java.base/sun.security.provider.PolicyFile$2.run(PolicyFile.java:335)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/sun.security.provider.PolicyFile.initPolicyFile(PolicyFile.java:335)
	at java.base/sun.security.provider.PolicyFile.init(PolicyFile.java:328)
	at java.base/sun.security.provider.PolicyFile.<init>(PolicyFile.java:281)
	at java.base/java.security.Policy.loadPolicyProvider(Policy.java:207)
	at java.base/java.security.Policy.getPolicyNoCheck(Policy.java:178)
	at java.base/java.security.ProtectionDomain.implies(ProtectionDomain.java:321)
	at java.base/java.security.ProtectionDomain.impliesWithAltFilePerm(ProtectionDomain.java:353)
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:450)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:897)
	at ch.qos.logback.core.util.Loader$1.run(Loader.java:48)
	at ch.qos.logback.core.util.Loader$1.run(Loader.java:45)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at ch.qos.logback.core.util.Loader.<clinit>(Loader.java:45)
	at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:73)
	at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:66)
	at ch.qos.logback.classic.spi.LogbackServiceProvider.initializeLoggerContext(LogbackServiceProvider.java:52)
	at ch.qos.logback.classic.spi.LogbackServiceProvider.initialize(LogbackServiceProvider.java:41)
	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:183)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:170)
	at org.slf4j.LoggerFactory.getProvider(LoggerFactory.java:455)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:441)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:390)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:416)
	at groovy.runtime.metaclass.CustomMetaClassCreationHandle.<clinit>(CustomMetaClassCreationHandle.java:53)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:315)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.installMetaClassCreationHandle(MetaClassRegistryImpl.java:191)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:139)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:94)
	at groovy.lang.GroovySystem.<clinit>(GroovySystem.java:37)
	at org.codehaus.groovy.runtime.InvokerHelper.<clinit>(InvokerHelper.java:71)
	at groovy.lang.GroovyObjectSupport.getDefaultMetaClass(GroovyObjectSupport.java:46)
	at groovy.lang.GroovyObjectSupport.<init>(GroovyObjectSupport.java:32)
	at groovy.lang.GString.<init>(GString.java:74)
	at groovy.lang.GString$1.<init>(GString.java:53)
	at groovy.lang.GString.<clinit>(GString.java:53)
	at nextflow.cli.Launcher.<clinit>(Launcher.groovy)
Caused by: java.nio.file.InvalidPathException: Malformed input or input contains unmappable characters: /Users/danielwodke/Dropbox/Universita��t/Semester_9/Operating Complex IT-Systems/serverless_scientific_workflow_poc/nextflow/work/6f/139cc4ff4d0770fdc3978aea906184
	at java.base/sun.nio.fs.UnixPath.encode(UnixPath.java:145)
	at java.base/sun.nio.fs.UnixPath.<init>(UnixPath.java:69)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:279)
	at java.base/java.io.FilePermission.<clinit>(FilePermission.java:207)
	... 44 more
